modules/apps/marketplace/marketplace-settings-web/src/main/java/com/liferay/marketplace/settings/web/internal/configuration/MarketplaceConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.marketplace.settings.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Keven Leone
+ */
+@ExtendedObjectClassDefinition(
+	generateUI = false, scope = ExtendedObjectClassDefinition.Scope.COMPANY
+)
+@Meta.OCD(
+	id = "com.liferay.marketplace.settings.web.internal.configuration.MarketplaceConfiguration",
+	localization = "content/Language", name = "marketplace-configuration-name"
+)
+public interface MarketplaceConfiguration {
+
+	public String accessToken();
+
+	public String clientId();
+
+	public String marketplaceSettings();
+
+	public String code();
+
+	public String refreshToken();
+
+	public long expiresIn();
+
+	public String redirect();
+
+	public String url();
```

Comment:
Sort alphabetically

Reply:
can you explain what exactly needs to be sort?

I mean, sort by type or variable name?

Reply:
Variable names need to be sorted alphabetically. It's the pattern we have for the *Configuration.java files

Reply:
Also, remember to add metatype specifications by using `@Meta.AD` annotation for each field

Reply:
Done